interface/usergroup/facility_admin.php

```diff
@@ -201,6 +201,14 @@ function displayAlert() {
                             <input type='entry' class='form-control form-control-sm' size='20' name='website' value="<?php echo attr($facility["website"]); ?>" />
                         </div>
                     </div>
+                    <div class="form-row my-2">
+                        <div class="col-2">
+                            <label for="weno_id" class='col-form-label col-form-label-sm'><?php echo xlt('Weno ID'); ?>:</label>
+                        </div>
+                        <div class="col">
+                            <input type="entry" class="form-controlform-control-sm" size="20" name="weno_id" value="<?php echo attr($facility["weno_id"]); ?>">
```

Comment:
looks like a typo, should be `form-control form-control-sm`

Reply:
@stephenwaite I could not figure out how to get the weno id to save. I know the data is being serialized and that where I am lost. I know it is being sent via ajax but could not track down where it is being unserialized. Do you know? Thanks!

Reply:
I figured it out. My dev was broken.